### PR TITLE
Use raw value if html color code

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
Fixes connected icon color of terminal

Issue:
Screenshot 2021-03-04 at 11 20 09

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

**Review by grolu — APPROVED** [review]
/lgtm

File: frontend/src/components/icons/IconBase.vue [hunk 1]
```diff
@@ -51,6 +51,9 @@ export default {
   computed: {
     iconColorCode () {
       const iconColor = this.iconColor
+      if (/^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$/.test(iconColor)) {
```
Comment (holgerkoser) [inline frontend/src/components/icons/IconBase.vue:54]:
```suggestion
      if (/^#([a-f0-9]{6}|[a-f0-9]{3})$/i.test(iconColor)) {
```

Comment (holgerkoser) [reply frontend/src/components/icons/IconBase.vue:54]: case insensitive regex

**Review by holgerkoser — APPROVED** [review]
/lgtm

File: frontend/src/components/icons/IconBase.vue (hunk 1 above)
Comment (grolu) [reply frontend/src/components/icons/IconBase.vue:54]: then we should do it in all places or even better move the regex to utils

Comment (petersutter) [reply frontend/src/components/icons/IconBase.vue:54]: moving it to utils makes sense

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**